src/fillTank.test.js

```diff
@@ -1,11 +1,132 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should return the full tank if the amount is not specified ', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 30);
+
+    expect(customer).toEqual({
+      money: 2040,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    });
+  });
+
+  it(`should be returned a full tank 
+  if the quantity is more than the tank can hold`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 8, 80);
+
+    expect(customer).toEqual({
+      money: 2744,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    });
+  });
+
+  it(`should return the liters in the tank
+  for which there is enough money`, () => {
+    const customer = {
+      money: 100,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 10, 20);
 
+    expect(customer).toEqual({
+      money: 0,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 18,
+      },
+    });
   });
 
-  // write tests here
+  it(`should return a rounded number if the request
```

Comment:
sounds really weird. you should use "non-integer number" or simply copy from readme " should round the poured amount by discarding number to the tenth part"

Reply:
thanks for the good check, all the wishes were added and corrected